ably-ios/ARTDataQuery+Private.h

```diff
@@ -16,4 +17,10 @@ ART_ASSUME_NONNULL_BEGIN
 
 @end
 
+@interface ARTRealtimeHistoryQuery ()
+
+@property (strong, readwrite) ARTRealtimeChannel *realtimeChannel;
```

Comment:
I really can't see and scenario in which this changes anything in practice, because the only way this property will be set is by `ARTRealtimeChannel.history` itself, and if you're calling that method obviously you _have_ the channel, and even if you drop the channel and hold to the query you can't do anything with the query but reuse it, which will change its `realtimeChannel`. But anyway, done at 547e39c.